src/classes/ERR_Handler_API.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>31.0</apiVersion>
```

Comment:
I think Nic has a pull request for moving to api 32, which will miss this new file.  do we up it now to avoid missing it?